hw/arm/prusa/stm32f407/stm32f2xx_crc.c

```diff
@@ -123,7 +124,14 @@ f2xx_crc_read(void *arg, hwaddr addr, unsigned int size)
     }
     switch(addr) {
     case R_CRC_DR:
-        return s->crc;
+        if (s->rcc && stm32_rcc_check_periph_clk(s->rcc, STM32_CRC)) 
+        {
+            return s->crc;
+        }
+        else
+        {
+            return 0xFFFFFFFFU;
```

Comment:
I could have mentioned it in the bug report, we need to invert the result (it's done in the implementation I wrote within the bug report). Function is returning 0xFFFFFFFF because CRC DR returns 0 :)